### PR TITLE
Add backend_backoff_max_delay flag to grpcwebproxy

### LINKED ISSUE #277: Configure backend backoff
During development, the 2 minute [default max backoff delay](https://github.com/grpc/grpc-go/blob/master/backoff.go#L31) is painfully long. It would be nice to be able to override this with a flag. The flag could map to the `WithBackoffMaxDelay` DialOption: https://godoc.org/google.golang.org/grpc#WithBackoffMaxDelay

I propose the flag be called `backend_backoff_max_delay` that would accept a duration string like "30s" (anything parseable with [time.ParseDuration](https://godoc.org/time#ParseDuration)).

I'd be happy to submit a PR.

### PR DESCRIPTION
**Pull request by fordhurley**
Closes #277.

### CODE REVIEW

File: go/grpcwebproxy/backend.go [hunk 1]
```diff
@@ -65,7 +71,10 @@ func dialBackendOrFail() *grpc.ClientConn {
 	} else {
 		opt = append(opt, grpc.WithInsecure())
 	}
+
 	opt = append(opt, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(*flagMaxCallRecvMsgSize)))
+	opt = append(opt, grpc.WithBackoffMaxDelay(*flagBackendBackoffMaxDelay))
```
Comment (johanbrandhorst) [inline go/grpcwebproxy/backend.go]: These two can just go in the same append call:

```go
opt = append(opt,
     grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(*flagMaxCallRecvMsgSize)),
     grpc.WithBackoffMaxDelay(*flagBackendBackoffMaxDelay),
)

**Comment by johanbrandhorst** [comment]
Nice work! I'll merge it after the tests have passed.

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Comment by fordhurley** [comment]
Awesome! Thank you!

**Pull request merged**